packages/lib/gpcircuits/circuits/bounds.circom

```diff
@@ -0,0 +1,67 @@
+pragma circom 2.1.8;
+
+include "circomlib/circuits/comparators.circom";
+include "circomlib/circuits/bitify.circom";
+include "gpc-util.circom";
+
+/**
+ * Module carrying out bounds checks on 64-bit unsigned integer values.
+ * 
+ * No overall enable flag, but can be disabled by checking whether 0
+ * lies in the interval [0, 1 << 64 - 1].
+ * 
+ * The values and bounds are assumed to be 64-bit unsigned integers
+ * but are not constrained here, cf. the comments regarding {@link
+ * InInterval}.
+ */
+template BoundsCheckModule(
+    // Number of values to be checked.
+    MAX_BOUNDS_CHECKS
+) {
+    // Values to be checked.
+    signal input comparisonValues[MAX_BOUNDS_CHECKS];
+
+    // Bounds that each corresponding value should lie within.
+    signal input bounds[MAX_BOUNDS_CHECKS][2];
```

Comment:
Variable naming/grouping discussion to have:
Would this be clearer as 2 separate variables for min and max?  The array of 2 seems to make things less obvious, though it. might have an advantage of fewer inputs at the top-level of the circuit.
Would the inclusive vs. exclusive question be clearer if these were `minValue` and `maxValue` rather than `bounds`?  To me, "min/max" imply inclusivity (because that's how the min(a,b) and max(a,b) functions work), while "bounds" is more ambiguous.